src/cloud-cli/cli.ts

```diff
@@ -70,6 +71,39 @@ program
     await getAppLogs(options.name, options.host, options.port);
   });
 
+const userdb = program
+  .command('userdb');
+   
+
+userdb
+  .command('create')
+  .argument('<string>', 'database name')
+  .option('-h, --host <string>', 'Specify the host', 'localhost')
+  .option('-p, --port <port>', 'Specify the port', '8080')
```

Comment:
Maybe safer/easier if we set the default values `localhost` and `8080` as variables on top of the file?